src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -102,308 +101,308 @@ public StudyController(StudyService studyService) {
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.studyExists(studyName, userId));
     }
 
-    @DeleteMapping(value = "/{userId}/studies/{studyName}")
+    @DeleteMapping(value = "/{userId}/studies/{studyUuid}")
```

Comment:
same

Reply:
same

Reply:
assertUserAllowed function needs that parameter